### PR TITLE
change meid to new_meid on change device api call

### PR DESCRIPTION
**Pull request by dougperez**

### CODE REVIEW

File: lib/conduit/sprint/decorators/base.rb [hunk 1]
```diff
@@ -28,6 +28,11 @@ def meid
       meid.upcase unless meid.blank?
     end
 
+   def new_meid
```
Comment (pedromartinez) [inline lib/conduit/sprint/decorators/base.rb]: Shouldn't this replace the method above?

Comment (dougperez) [inline lib/conduit/sprint/decorators/base.rb]: We still use the meid for other api calls. I could move this to the ChangeDevice decorator instead of Base.

Comment (dougperez) [inline lib/conduit/sprint/decorators/base.rb]: Moved out of base.

**Comment by pedromartinez** [comment]
👍

**Pull request merged**